spanner/api/Spanner.Samples/ReadWriteWithTransactionCoreAsync.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_read_write_transaction_core]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class ReadWriteWithTransactionCoreAsyncSample
+{
+    public async Task<int> ReadWriteWithTransactionCoreAsync(string projectId, string instanceId, string databaseId)
+    {
+        // This sample transfers 20,000 from the MarketingBudget
+        // field of the second Album to the first Album. Make sure to run
+        // the Add Column and Write Data To New Column samples first,
+        // in that order.
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        decimal transferAmount = 20000;
```

Comment:
The amount in the existing sample has another 0.

https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/201ca45e06992d803802833cbfc05d2be768d8d0/spanner/api/Spanner/Program.cs#L1175